picasso/src/test/java/com/squareup/picasso/UtilsTest.java

```diff
@@ -119,16 +114,4 @@
     int id = Utils.getResourceId(res, request);
     assertThat(id).isEqualTo(RESOURCE_ID_1);
   }
-
-  @Test @Config(reportSdk=GINGERBREAD) public void useOkHttpByDefault() throws Exception {
-    Downloader downloader = Utils.createDefaultDownloader(Robolectric.application);
-    assertThat(downloader).isInstanceOf(OkHttp3Downloader.class);
-  }
-
-  @Test @Config(reportSdk=FROYO) public void noOkHttpInFroyo() throws Exception {
-    Downloader downloader = Utils.createDefaultDownloader(Robolectric.application);
-    assertThat(downloader)
-        .isNotInstanceOf(OkHttpDownloader.class)
-        .isNotInstanceOf(OkHttp3Downloader.class);
-  }
```

Comment:
Alright. So leave these removed? And is there anything else from this test I should be removing?